src/cascadia/TerminalSettingsEditor/ColorToHexConverter.cpp

```diff
@@ -20,18 +20,11 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         return winrt::box_value(hex);
     }
 
-    Foundation::IInspectable ColorToHexConverter::ConvertBack(Foundation::IInspectable const& value,
+    Foundation::IInspectable ColorToHexConverter::ConvertBack(Foundation::IInspectable const& /*value*/,
                                                               Windows::UI::Xaml::Interop::TypeName const& /* targetType */,
-                                                              Foundation::IInspectable const& /* parameter */,
+                                                              Foundation::IInspectable const& /*parameter*/,
                                                               hstring const& /* language */)
     {
-        auto str = winrt::unbox_value<winrt::hstring>(value);
-        std::wstring hex{ str.data() };
-        Windows::UI::Color newColor;
-        newColor.A = base::checked_cast<uint8_t>(std::stoi(hex.substr(1, 2), nullptr, 16));
-        newColor.R = base::checked_cast<uint8_t>(std::stoi(hex.substr(3, 2), nullptr, 16));
-        newColor.G = base::checked_cast<uint8_t>(std::stoi(hex.substr(5, 2), nullptr, 16));
-        newColor.B = base::checked_cast<uint8_t>(std::stoi(hex.substr(7, 2), nullptr, 16));
-        return winrt::box_value(newColor);
+        throw hresult_not_implemented();

```

Comment:
Why? If it was there and working.... why remove it?

Reply:
It was pretty naive and needed some more error handling, and after simplifying the ColorScheme UI to only have a Color picker, I didn't need the `ConvertBack` of this converter anymore.

Reply:
alright that's fine.